tests/test_update_sensor.py

```
@@ -1079,33 +1227,53 @@ def set_attr(key: str, value: object) -> None:
     ],
 )
 async def test_calculate_travel_distance_variants(
-    mock_hass, mock_config_entry, sensor, mode, blank_attr, expected_direction, expect_mi
-):
+    mock_hass: MagicMock,
+    mock_config_entry: MockConfigEntry,
+    sensor: MockSensor,
+    mode: str,
+    blank_attr: str,
+    expected_direction: str | None,
+    expect_mi: bool,
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)